notifications/2024-12-05_CHG0065024.md

```diff
@@ -0,0 +1,27 @@
+Hi @all, your attention is needed! 
+
+**What is happening?**
+
+CHG0065024 CITZ MCS EMERALD - Clean up AVI TLS profiles
+
+Do some manual cleanup of intermediate and root TLS certificates in the AVI load balancer
+
+**When?**
+
+Starting at Thursday, Dec 12 at 6:00 AM and finishing by 8:00 AM
```

Comment:
wrong date? RFC approval email says December 5th

Reply:
Wrong date. This is why we get reviews, thanks!